src/ripple/ledger/impl/View.cpp

```diff
@@ -713,13 +713,10 @@ dirAdd (ApplyView& view,
     std::function<void (SLE::ref)>          fDescriber,
     beast::Journal j)
 {
-    if (view.rules().enabled(featureSortedDirectories))
-    {
-        if (strictOrder)
-            return view.dirAppend(dir, uLedgerIndex, fDescriber);
-        else
-            return view.dirInsert(dir, uLedgerIndex, fDescriber);
-    }
+    if (strictOrder)
+        return view.dirAppend(dir, uLedgerIndex, fDescriber);
+    else
+        return view.dirInsert(dir, uLedgerIndex, fDescriber);
```

Comment:
Consider eliminating the else statement

`if (strictOrder)
    return view.dirAppend(dir, uLedgerIndex, fDescriber);
return view.dirInsert(dir, uLedgerIndex, fDescriber);`

Reply:
Ah never mind, I see you did that in a later commit.